lambda/getProductsById.ts

```diff
@@ -0,0 +1,33 @@
+import { mockProducts } from './mockData';
+
+const headers = {
+  "Content-Type": "text/plain",
+  "Access-Control-Allow-Origin" : "*",
+};
+
+export const handler = async (event: any) => {
+  try {
+    const product = mockProducts.find(({ id }) => id === event.pathParameters.productId);
+
+    if (!product) {
+      return {
+        statusCode: 404,
+        headers,
+        body: 'Not found',
+      };
+    }
+
+    return {
+      statusCode: 200,
+      headers,
```

Comment:
it would be better to set `"content-type": "application/json"` for the 200 OK response.